api/proto/teleport/trust/v1/trust_service.proto

```diff
@@ -35,3 +37,17 @@ message GetCertAuthorityRequest {
   // Whether the private key should be included in the response.
   bool include_key = 3;
 }
+
+// Request for GetCertAuthorities
+message GetCertAuthoritiesRequest {
+  // Type of certificate authority.
+  string type = 1;
+  // Whether the private key should be included in the response.
```

Comment:
Should we add something like this, to avoid another SAMLIdP CA situation?

```suggestion
  // Whether the private key (if available) should be included in the response. Should only be used for administrative tasks.
```